package.json

```diff
@@ -11,10 +11,17 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build && yarn eclint && yarn test",
+    "build": "yarn prettier && react-scripts build && yarn eclint && yarn test",
```

Comment:
I think that build should not modify files, it should verify that the files are pretty already.

`prettier-check` package is useful for that. See https://github.com/MakingSense/doppler-feature-toggle-js/blob/master/package.json#L24